Marsiple/App/New Group/PhotoCollectionViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PhotoCollectionViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class PhotoCollectionViewController: UICollectionViewController {
+    private let photos: [UIImage]
```

Comment:
Lets add a `NOTE:` here, saying that this is just test code, to be replaced with `URL`s in the future.